.github/workflows/python_release.yml

```diff
@@ -35,14 +35,30 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: Publish to pypi (without sdist)
-        uses: messense/maturin-action@main
+        uses: messense/maturin-action@v1
         env:
           MATURIN_PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
         with:
           target: ${{ matrix.target }}
           command: publish
           args: -m python/Cargo.toml --no-sdist
 
+  release-pypi-mac-universal2:
+    needs: validate-release-tag
+    name: PyPI release on Mac universal 2
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Publish to pypi (without sdist)
+    uses: messense/maturin-action@v1
+    env:
+        MATURIN_PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
+    with:
+        target: ${{ matrix.target }}
+    command: publish
+    args: -m python/Cargo.toml --no-sdist --universal2
```

Comment:
Is this indentation correct? are these all supposed to be part of the step? And is `command` and `args` supposed to be under `with`?

Reply:
Oh, thanks; good catch!

Indeed, I need to reformat the file 👍